src/backend/utils/storage.js

```diff
@@ -63,6 +63,15 @@ module.exports = {
     return redis.set(key, reason);
   },
 
+  removeFeed: async id => {
+    const key = createFeedKey(id);
+    await redis
```

Comment:
Probably we should remove the posts here at the redis level when we remove the feed.